vault/request_handling.go

```diff
@@ -171,7 +171,7 @@ func (c *Core) handleRequest(req *logical.Request) (retResp *logical.Response, r
 		if errType != nil {
 			retErr = multierror.Append(retErr, errType)
 		}
-		return logical.ErrorResponse(ctErr.Error()), nil, retErr
+		return logical.ErrorResponse(ctErr.Error()), auth, retErr
```

Comment:
It is necessary to get it to log the info in the response entry as well.  In the case of an error, including the ACL error scenario I'm trying to address here, the auth object from this return statement then gets passed to `c.auditBroker.LogResponse` on line 107.